### ISSUE TITLE
question about the MACs of nn.BatchNorm2d

### ISSUE THREAD

**Issue by DaMiBear** [Lyken17/pytorch-OpCounter#196]
Hi, happy new year! 
I'm confused about the method of calculating nn.BatchNorm2d MACs.
https://github.com/Lyken17/pytorch-OpCounter/blob/43c064afb71383501e41eaef9e8c8407265cf77f/thop/vision/basic_hooks.py#L60-L69
https://github.com/Lyken17/pytorch-OpCounter/blob/43c064afb71383501e41eaef9e8c8407265cf77f/thop/vision/calc_func.py#L43-L45
In my opinion: in `calculate_norm(input_size)`, the `2 * input_size` already means the MACs of subtract(mean), divide(var), mul(weight) and add(bias). But why is the `flops`(w.r.t MACs) multiplied by 2 again in the next?
